tests/SDAccel/hls_with_cppkernel_opencl_kernel_ap_vector_add_args.cpp

```diff
@@ -0,0 +1,88 @@
+/* Simple OpenCL vector addition using SYCL in OpenCL interoperability
```

Comment:
Explain more what the 2 new examples do, since it is no longer a simple copy-paste...